### ISSUE TITLE
Click and drag in line number gutter does not select lines

### ISSUE THREAD

**Issue by zwierko** [Komodo/KomodoEdit#3818]
### Short Summary
When clicking and dragging in the line number gutter, only the first line is selected instead of multiple lines. This used to be the behavior in previous versions of Komodo, and is standard behavior in most other editors.

### Steps to Reproduce
Click and drage the mouse in the gutter next to lines your would like selected.

### Expected results
Multiple lines are selected.

### Actual results
Only the first line is ever selected.

### Platform Information

Komodo IDE
Version 12.0.0
Operating System (and version): Windows 10 Enterprise 2016 LTSB

**Comment by th3coop** [comment]
@zwierko I've honestly never heard of this.  Which version of Komodo do you recall using this in?  It'll help me track down what changed.

**Comment by th3coop** [comment]
@zwierko nvm, looks like it work sin Komodo 10.  As a workaround, is anything preventing you from clicking just to the right, in the editor area instead of the line number, and doing the same action?  This should achieve the same thing.

**Comment by zwierko** [comment]
That's what I have been doing. I just wanted to report it in hopes the behavior would be fixed back to what it was. It also makes Komodo consistent with most other popular IDEs and editors. It's a minor thing though, so I appreciate your time.

**Comment by th3coop** [comment]
> I just wanted to report it in hopes the behavior would be fixed back to what it was.

Absolutely.  I appreciate the report.  I just wanted to make sure you're not blocked.  Thanks for the report, @zwierko.
